pact/yaml/testnet/ns.yaml

```diff
@@ -1,8 +1,9 @@
-codeFile: ../ns.pact
+codeFile: ../../ns.pact
 signers:
-  - public: dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e
+  - public: ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6
   - public: 368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca
 data:
+  ns: "kswap"
   swap-ns-user:
     keys:
       - dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e
```

Comment:
I see that `dfb16b13e4032a6878fd98506b22cb0d6e5932c541e656b7ee5d69d72e6eb76e` was switched out with `ef711514b5a248ac16ffd5cc94511dfccbd89005e9737343dfbb3743c8a63ca6`, but I still see it in `keys`. Is that intentional?

Reply:
Yes, this is the consequence of a different person (me) running the tx, but the keyset still having me and the previous runner (HK) in the keyset.